cmd/account/reset.go

```diff
@@ -114,25 +121,46 @@ func (o *resetOptions) run() error {
 			}
 		}
 	}
-
 	account, err := k8s.GetAWSAccount(ctx, o.kubeCli, o.accountNamespace, o.accountName)
 	if err != nil {
 		return err
 	}
-
+	//get accountID for reset
+	var accountId string
+	accountId = account.Spec.AwsAccountID
+	//	payerAccount = account.Spec.IAMUserSecret
 	// reset fields in spec
-	account.Spec.ClaimLink = ""
-	account.Spec.ClaimLinkNamespace = ""
-	account.Spec.IAMUserSecret = ""
+	//	account.Spec.ClaimLink = ""
+	//	account.Spec.ClaimLinkNamespace = ""
+	//	account.Spec.IAMUserSecret = ""
+	//awsClient, err := awsprovider.NewAwsClient("osd-staging-2", "us-east-1", "")
+	awsClient, err := o.getAwsClient("aws-account-operator-credentials", "aws-account-operator")
```

Comment:
Can you either move the login into this function, or rename the function to match what's happening (getAwsAccountOperatorSecret/Client). I also think using an &Client would be cleaner.

Reply:
Here are my thoughts.
1. Get the Secret from the namespace
2. get the Access key and Secret access key from the secret data
3. Call awsprovider.NewAwsClientWithInput similar to https://github.com/openshift/osdctl/blob/e5b4ae5c11a245ebed83ee152bfc35ea9b0fcc29/cmd/account/clean-velero-snapshots.go#L78